app/js/app/services/notifications.js

```diff
@@ -0,0 +1,241 @@
+angular.module('app.services').factory('notifications',function ($window, device, notificationsData, onNotificationGCM, onNotificationAPN, serverConfig) {
```

Comment:
We don't have notifications in web app, but had discussed these being e-mail or desktop notifications instead. I think Chrome-based Desktop notifications makes sense, but may want to remove this file.